packages/react-core/src/components/Label/Label.tsx

```diff
@@ -59,6 +59,8 @@ export interface LabelProps extends React.HTMLProps<HTMLSpanElement> {
   href?: string;
   /** Flag indicating if the label is an overflow label */
   isOverflowLabel?: boolean;
+  /** On click callback. If present, label will be clickable. */
```

Comment:
Nit: "Callback for when the label is clicked." We could omit the "label will be clickable" portion I think,

Or per my above comment, "Callback for when the label is clicked. This should not be passed in if the href prop is also passed in." (the href prop could just have "This should not be passed in if the onLabelClick prop is also passed in."

Reply:
+1

Reply:
+1 from me as well. Also name conflict forces us to not use `onClick` as the prop name? I think that's the prop name I would expect for this as a consumer.